### ISSUE TITLE
Centralize configuration

### ISSUE THREAD

**Issue by stormofice** [mensatt/frontend-old#106]
Here are all occurrences of static values, which may be better kept in a separate configuration file:
- [x] Locations/Backend url
https://github.com/mensatt/frontend/blob/1ed0682580b2309c733a761f65b5a2ed3cdc4ef7/src/apollo/cache.ts#L38-L47
- [x] Backend url
https://github.com/mensatt/frontend/blob/1ed0682580b2309c733a761f65b5a2ed3cdc4ef7/src/apollo/client.ts#L14
- [ ] Available languages
https://github.com/mensatt/frontend/blob/1ed0682580b2309c733a761f65b5a2ed3cdc4ef7/src/components/navigation/language-switcher/LanguageSwitcher.tsx#L9-L12
- [ ] Protected routes
https://github.com/mensatt/frontend/blob/1ed0682580b2309c733a761f65b5a2ed3cdc4ef7/src/hocs/route-guard/RouteGuard.tsx#L20-L21
- [x] Backend url
https://github.com/mensatt/frontend/blob/aac4aae1a2d06b7a794f5db7c652e50aad44e6f9/src/components/occurrence/Occurrence.tsx#L35-L39
- [ ] Max image file size
https://github.com/mensatt/frontend/blob/aac4aae1a2d06b7a794f5db7c652e50aad44e6f9/src/components/occurrence/review-modal/ReviewModal.tsx#L32

**Comment by GamerBene19** [comment]
I'm currently working on this issue in #188. I came to the conclusion that it only makes sense to extract _shared_ values (currently only API-URLs are shared). 
Thoughts?

**Comment by stormofice** [comment]
I agree, I think only reusable, shared values should be accessible this way.
